apps/rush/src/RushVersionSelector.ts

```diff
@@ -30,7 +30,7 @@ export class RushVersionSelector {
     const isLegacyRushVersion: boolean = semver.lt(version, '4.0.0');
     const expectedRushPath: string = path.join(this._rushGlobalFolder.nodeSpecificPath, `rush-${version}`);
 
-    const installMarker: _LastInstallFlag = new _LastInstallFlag(expectedRushPath, {
+    const installMarker: _FlagFile = new _FlagFile(expectedRushPath, 'last-install', {
```

Comment:
Note: This implies the `last-install.flag` file is now controlled by "rush" instead of "rush-lib".  Previously, it's an internal logic inside "rush-lib".

Reply:
in this case, rush is using it's own version of "last-install.flag" to keep track of something different then rush-lib. The LastInstallFlag inside rush-lib is very specific to the installation of workspaces, whereas the flag being used here is much simpler (should be thought of as independent from the last install flag used in rush-lib)